src/core_plugins/markdown_vis/public/github_markdown.less

```diff
@@ -197,6 +197,14 @@
   margin-bottom: 0;
 }
 
+// Kibana sets list-style: none. Need to restore list styles for markdown
+.markdown-body ul li {
```

Comment:
Any reason not to apply the list style to the whole list, i.e. `ul`, not the list item?

Reply:
no reason. I will apply the style to the list element